### PR TITLE
Feature/seccomp

### PR DESCRIPTION
**Pull request by amitmalav**
Added libseccomp package

### CODE REVIEW

File: jobs/service-fabrik-deployment-hooks/templates/bin/service-fabrik-deployment-hooks_ctl.erb [hunk 1]
```diff
@@ -6,6 +6,22 @@ source /var/vcap/packages/bosh-helpers/ctl_setup.sh 'service-fabrik-deployment-h
...
+pushd ${HOME}/packages/libseccomp
+./configure
+make
+make install
+# set linkings
+ldconfig -v
+popd
+
+# compile add_seccomp_filters script
+pushd ${HOME}/packages/service-fabrik-deployment-hooksdeployment_hooks/bin/seccomp
```
Comment (kamath-prasad) [inline jobs/service-fabrik-deployment-hooks/templates/bin/service-fabrik-deployment-hooks_ctl.erb]: is a "/" missing here ?

should it be `pushd ${HOME}/packages/service-fabrik-deployment-hooks/deployment_hooks/bin/seccomp` ?

Comment (amitmalav) [reply jobs/service-fabrik-deployment-hooks/templates/bin/service-fabrik-deployment-hooks_ctl.erb]: My Bad! Done!

File: packages/libseccomp/spec [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+name: libseccomp
+dependencies: []
+files:
+- libseccomp/libseccomp-2.3.3.tar.gz
```
[the remark this answers is not in the document]
Comment (amitmalav) [reply packages/libseccomp/spec]: Done. Uploaded blobs.

File: jobs/service-fabrik-deployment-hooks/templates/bin/service-fabrik-deployment-hooks_ctl.erb [hunk 3]
```diff
@@ -6,6 +6,22 @@ source /var/vcap/packages/bosh-helpers/ctl_setup.sh 'service-fabrik-deployment-h
...
 export PID_FILE=${RUN_DIR}/service-fabrik-deployment-hooks.pid
 export PACKAGE_DIR=${HOME}/packages/service-fabrik-deployment-hooks
+export SECCOMP_CMD=${PACKAGE_DIR}/deployment_hooks/bin/seccomp/seccomp
+
+# installing libseccomp here because .so files are needed at runtime for seccomp executable 
+# installing libseccomp
+pushd ${HOME}/packages/libseccomp
+./configure
+make
+make install
```
[the remark this answers is not in the document]
Comment (amitmalav) [reply jobs/service-fabrik-deployment-hooks/templates/bin/service-fabrik-deployment-hooks_ctl.erb]: Changed the approach of installation.
Now installation of `libseccomp` happens in packaging step and installation path is set to `/var/vcap/packages/libseccomp/libseccomp_installation`
Compilation of `add_seccomp_filters.c` also happens in packaging step with proper linking to libseccomp object files.
In ctrl script now we just create the necessary links and cache.
Now package as well as installation will get removed in next deploy.

[the remark this answers is not in the document]
Comment (sauravmndl) [reply jobs/service-fabrik-deployment-hooks/templates/bin/service-fabrik-deployment-hooks_ctl.erb]: AWESOME!!

**Pull request merged**